tests/unit/dataset/_utils/_archives_test.py

```diff
@@ -0,0 +1,591 @@
+# Copyright (c) 2022-2025 The pymovements Project Authors
```

Comment:
are these tests just copies or should I check them?

Reply:
the tests are just copied